### ISSUE TITLE
SC 2.4.11 - Visual focus does not have enough color contrast

### ISSUE THREAD

**Issue by mpaiva** [mpaiva/CRDC-DSRC#4]
[SC 2.4.11](https://www.w3.org/TR/WCAG22/#focus-appearance-minimum)

When the keyboard focus indicator is visible, an area of the focus indicator meets all the following:

- is at least as large as the area of a 2px thick perimeter of the unfocused component or sub-component, and
- has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.

![image](https://github.com/mpaiva/CRDC-DSRC/assets/781670/936fa27b-e4ab-44ea-8f1b-a4d410f58b8b)

![image](https://github.com/mpaiva/CRDC-DSRC/assets/781670/b0da600a-996f-43e4-a321-9e75511e3f30)

## How to fix
Comment the following CSS line:
`    /* outline: .1rem solid rgba(176, 217, 222, .4941176471);`

The result should be as the image below:

![image](https://github.com/mpaiva/CRDC-DSRC/assets/781670/fa392b7e-0e18-4acd-894d-efa11e2eccbb)
